themes/default/content/resources/getting-started-infrascructure-as-code-java/index.md

```diff
@@ -0,0 +1,94 @@
+---
+# Name of the webinar.
+title: "Get Started with Infrastructure as Code on Azure with Java"
```

Comment:
I changed this from "Getting Started" to "Get Started" since the description was 2 characters too long. Let me know if this is ok.